tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py38, flake8, mypy, docs, black, bandit
+envlist = py38, flake8, mypy, docs, black, bandit, pyspelling
```

Comment:
the environment here needs to match the one below, if you put `documents` then this needs to be `documents` if we change to `spelling` then that it is.

Reply:
Reverted to original tox.in as it is not practical to configure for the local environment. To configure and run it in its own local environment instructions can be at found https://facelessuser.github.io/pyspelling/. I will add this to documentation.